src/react/2-intro-to-jsx.md

```diff
@@ -152,9 +152,9 @@ Will output the following HTML:
 <div>Sibling 2</div>
 ```
 
-Note how the fragment components don't have any corresponding DOM output. This would also be true if we had used `<React.Fragment>` instead; they are the same.
+Note how the fragment components don't have any corresponding DOM output. This would also be true if `<React.Fragment>` was used instead; they are the same.
 
-## JavaScript Interpolation
+## JavaScript's JSX Interpolation
```

Comment:
JavaScript is what's being interpolated by JSX, not the other way around. I think it was good as it ways: Section is JSX, subsection is interpolating JavaScript within JSX.